createmobilespec/createmobilespec.js

```diff
@@ -413,7 +419,7 @@ function verifyNpmLinkOf(linkedModule, installedModule) {
 }
 
 if (!argv.skiplink) {
-    console.log("Checking if you are using master branch of tools");
+    console.log("Checking if you are using master branch of tools (js, lib, plugman, cli)");
```

Comment:
I think we have no need to actually check js, added a commit to PR #174 to update this message there.